apps/site/assets/js/event-page-setup.js

```diff
@@ -62,9 +62,11 @@ export function setupEventsPage() {
   if (activeMonthElement) activeMonthElement.scrollIntoView();
 
   // add event listener to navigate to page on dropdown select
-  const monthSelect = control.querySelector(".m-event-list__select");
-  monthSelect.addEventListener("change", ({ target }) => {
-    window.location.assign(target.value);
+  const dateSelects = eventsHubPage.querySelectorAll(".m-event-list__select");
+  dateSelects.forEach(select => {
```

Comment:
[IE is quirky about `forEach`](https://stackoverflow.com/a/55741748), can we convert to good ol' `for`? 😞